### PR TITLE
feat(config): add ABUS SHHA10000 configuration

### PR DESCRIPTION
**Pull request by Technikte**
Add the ABUS smart switch plug, like mentioned here https://github.com/zwave-js/node-zwave-js/issues/1875
Work In Progress at the moment

### CODE REVIEW

File: packages/config/config/devices/0x0403/shha10000.json [hunk 1]
```diff
@@ -16,6 +16,240 @@
...
+				}
+			]
+		},
+		"17": {
+			"label": "Threshold Power Report Percentage",
+			"description": "The threshold value serves as an additional condition of the time-based report",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 127,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0403/shha10000.json:133]: 127%, interesting 🤔

File: packages/config/config/devices/0x0403/shha10000.json [hunk 2]
```diff
@@ -16,6 +16,240 @@
...
+					"value": 0
+				},
+				{
+					"label": "Reset to default Values",
+					"value": 85
+				}
+			]
+		},
+		"101": {
+			"label": "Power Report (Watt)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0403/shha10000.json]: I think this should have "Interval" in the label, maybe like this
```suggestion
			"label": "Power (Watt) Report Interval",
```

Comment (AlCalzone) [reply packages/config/config/devices/0x0403/shha10000.json]: or (Watt) at the end. Same goes for the next params.

File: packages/config/config/devices/0x0403/shha10000.json [hunk 3]
```diff
@@ -16,6 +16,240 @@
...
+					"value": 0
+				},
+				{
+					"label": "Locked",
+					"value": 1
+				}
+			]
+		}
+	},
 	"metadata": {
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0403/shha10000.json:254]: If you don't have any instructions, remove the metadata.

File: packages/config/config/devices/0x0403/shha10000.json [hunk 4]
```diff
@@ -16,6 +16,240 @@
...
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED is permanently on when device on",
+					"value": 0
+				},
+				{
+					"label": "LED is on for 5 seconds when device changing to on",
+					"value": 1
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0403/shha10000.json]: Labels should be a bit shorter

File: packages/config/config/devices/0x0403/shha10000.json [hunk 5]
```diff
@@ -16,6 +16,240 @@
...
+			]
+		},
+		"16": {
+			"label": "Threshold Power Report Value",
+			"description": "The threshold value serves as an additional condition of the time-based report",
+			"valueSize": 2,
+			"unit": "Watt",
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 50,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0403/shha10000.json:115]: add `"unsigned": true`

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by Technikte** [comment]
@AlCalzone @blhoward2 I hope it looks okay, its ready for review from my side.

File: packages/config/config/devices/0x0403/shha10000.json (hunk 3 above)
Comment (Technikte) [reply packages/config/config/devices/0x0403/shha10000.json:254]: Already done

File: packages/config/config/devices/0x0403/shha10000.json (hunk 5 above)
Comment (Technikte) [reply packages/config/config/devices/0x0403/shha10000.json:115]: Already done

**Comment by AlCalzone** [comment]
Seems I was a bit quick

**Comment by Technikte** [comment]
> 
> 
> Seems I was a bit quick

Yeah kinda :D , was in a draft state when you commented and then shortly after that I pushed the last commit and marked ready for review.

File: packages/config/config/devices/0x0403/shha10000.json (hunk 1 above)
Comment (Technikte) [reply packages/config/config/devices/0x0403/shha10000.json:133]: The manual says.. quite uncommonm but I could not find any other info..
grafik

File: packages/config/config/devices/0x0403/shha10000.json (hunk 2 above)
Comment (Technikte) [reply packages/config/config/devices/0x0403/shha10000.json]: Completed

File: packages/config/config/devices/0x0403/shha10000.json (hunk 4 above)
Comment (Technikte) [reply packages/config/config/devices/0x0403/shha10000.json]: Completed

File: packages/config/config/devices/0x0403/shha10000.json (hunk 1 above)
Comment (Technikte) [reply packages/config/config/devices/0x0403/shha10000.json:133]: Its the same in the english manual..

> -128 –127 = Available Settings

So I would say it is ok even if it is a bit weird.

Comment (AlCalzone) [reply packages/config/config/devices/0x0403/shha10000.json:133]: WTF. Well, then it is okay.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**